client/packages.go

```diff
@@ -24,12 +24,12 @@ import (
 	"fmt"
 )
 
-// Package represents a Snap package
-type Package struct {
+// Snap represents a Snap snap
```

Comment:
Ouch. :)

`// SnapInfo holds the data for a snap as obtained from the server.`

?

The `Info` suffix to highlight the difference between that and stuff from the `snap` directory?  Just thinking out loud for now. Happy to brainstorm on this online when you have a moment.